src/liteclient.h

```diff
@@ -40,7 +40,7 @@ class LiteClient {
 
   bool checkForUpdatesBegin();
   void checkForUpdatesEnd(const Uptane::Target& target);
-  bool complete(bool finalize);
+  bool complete();
```

Comment:
I like this change. While we are at this, maybe its time to give this method a better name since its now public. e.g. `complete_previous_install` or something.

Reply:
Jorge wanted something shorter :)... I would prefer to stick to the aktualizr's terms, which is finalize_installation :).

Reply:
I think `finalize_install{ation}` actually makes more sense also.

Reply:
renamed.